tsconfig.json

```diff
@@ -11,8 +11,13 @@
     "jsx": "react",
     "jsxFactory": "h",
     "allowJs": false,
-    "baseUrl": "."
-  },
+    "baseUrl": ".",
+    "paths": {
+      "codecs/*": ["src/codecs/*"],
+      "lib/*": ["src/lib/*"],
+      "@codecs/*": ["codecs/*"]
```

Comment:
We should come up with something better than `"@codecs"` vs `"codecs"`.

Having a prefix for all our aliases seems good though.

Reply:
Maybe rename `codecs` to `wasm-codecs` and use `@` as a prefix for all the aliases?

Reply:
Using `@` seems contentious here, since `@aaa/bbb` is a valid npm package name and bare specifier. I have seen most folks use `~` as a prefix, since it has no other useful meaning within a project.